### ISSUE TITLE
tslint test does not handle CRLF when lint failures span multiple lines

### ISSUE THREAD

**Issue by jamcoupe** [palantir/tslint#4170]
### Bug Report

- __TSLint version__:
- __TypeScript version__:
- __Running TSLint via__: (pick one) CLI / Node.js API / VSCode / grunt-tslint / Atom / Visual Studio / etc

#### TypeScript code being linted

```ts
// code snippet
```

with `tslint.json` configuration:

```json

```
